src/shipit_code_coverage_crawler/default.nix

```diff
@@ -0,0 +1,36 @@
+{ releng_pkgs
+}: 
+
+let
+
+  inherit (releng_pkgs.lib) mkBackend fromRequirementsFile filterSource;
```

Comment:
We have to replace mkBackend with mkBot (example usage in src/shipit_code_coverage/default.nix).